Sources/Result/ValidationResult.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+
+/// Validation result type.
+///
+/// - success: Validation was succesfull
+/// - failure: Validation failed. Contains `ValidationResult.Summary` that details the cause for failure
+public enum ValidationResult {
+    case success
```

Comment:
I noticed that you choose to rename the `valid` case to `success` and the `invalid` case to `failure`.  At some point, I did the rename in the opposite direction (from `success`/ `failure` to `valid`/ `invalid`).

I think I missed something and it would help to understand your reasoning? Like, when using the API, how should it be read? Thx!

Reply:
My main reasoning was to not interfere with Swift 5 new Result type. From my experience having same class names in different modules becomes cumbersome to use, since you have to explicitly specify which module you actually want to use. 
Take this as an example:
You declare an enum in your framework called `Error` and make it conform to `Swift.Error`.
When you start to use that type in application you would need to explicitly specify `<Module_name>.Error` or `Swift.Error`.

Renaming cases is also related to Swift's new `Result` type, just to be consistent. I did try to use Result type, but it needs a value for success to work properly.

Reply:
👍